README.md

```diff
@@ -1,29 +1,3 @@
-# EzHr-Chatbot LLM-based Assistant
+# Chatbot
 
-This is the LLM-based assistant for the EzHr-Chatbot project. It makes use of RAG and LlamaIndex to provide a conversational interface for the users.
-
-## Authors
-
-- Meow
-
-## Components
-
-### API Server
-
-This is the API server for the EzHr-Chatbot LLM-based assistant. It is built using FastAPI and LlamaIndex for LLM framework coding.
-
-For developing and testing purposes, we can use the Docker compose to run the API server.
-
-```bash
-# Move to the docker_compose directory
-cd deployment/docker_compose
-
-# Create the environment file
-cp .env.example .env
-
-# Build the docker image
-docker-compose build
-
-# Run the docker container
-docker-compose up -d
-```
\ No newline at end of file
+boi boi boi
```

Comment:
What is this ?

Reply:
Fuck you
I am trying to sign my commit
Please ignore

Reply:
I think your behavior right now is not professional. Please test on your local github, not a pull request for everyone to see. Thank you. Good night.

cc @tuantran0910

Reply:
> Member

I do agree with you. We should punish him.

Reply:
No.
I need boi q to support us.